src/core/lib/gpr/sync_posix.cc

```diff
@@ -181,7 +190,9 @@ int gpr_cv_wait(gpr_cv* cv, gpr_mu* mu, gpr_timespec abs_deadline) {
 #endif  // GPR_LINUX
     abs_deadline_ts.tv_sec = static_cast<time_t>(abs_deadline.tv_sec);
     abs_deadline_ts.tv_nsec = abs_deadline.tv_nsec;
+
```

Comment:
Nit: please remove these blank lines that were added

Reply:
Reverted.